python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 1161
   - name: u_centroid_yErr
     "@id": "#Object.u_centroid_yErr"
     datatype: float
     description: 1-sigma uncertainty on y position. Measured on u-band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 1162
   - name: u_decl
     "@id": "#Object.u_decl"
     datatype: double
     description: Position in Declination. Measured on u-band.
     fits:tunit: deg
     ivoa:ucd: pos.eq.dec
+    tap:principal: 0
+    tap:column_index: 1002
   - name: u_extendedness
     "@id": "#Object.u_extendedness"
     datatype: double
     description: Set to 1 for extended sources, 0 for point sources. Measured on u-band.
     fits:tunit:
+    tap:principal: 1
+    tap:column_index: 1163
   - name: u_extendedness_flag
     "@id": "#Object.u_extendedness_flag"
     datatype: boolean
     description: Set to 1 for any fatal failure. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1164
   - name: u_free_cModelFlux
     "@id": "#Object.u_free_cModelFlux"
     datatype: double
     description: Flux from the final cmodel fit. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1165
   - name: u_free_cModelFluxErr
     "@id": "#Object.u_free_cModelFluxErr"
     datatype: double
     description: Flux uncertainty from the final cmodel fit. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1166
   - name: u_free_cModelFlux_flag
     "@id": "#Object.u_free_cModelFlux_flag"
     datatype: boolean
     description: Flag set if the final cmodel fit (or any previous fit) failed. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1167
   - name: u_free_cModelFlux_inner
     "@id": "#Object.u_free_cModelFlux_inner"
     datatype: double
     description: Flux within the fit region, with no extrapolation. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1168
   - name: u_free_psfFlux
     "@id": "#Object.u_free_psfFlux"
     datatype: double
     description: Flux derived from linear least-squares fit of PSF model. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1169
   - name: u_free_psfFluxErr
     "@id": "#Object.u_free_psfFluxErr"
     datatype: double
     description: Flux uncertainty derived from linear least-squares fit of PSF model. Measured on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1170
   - name: u_free_psfFlux_flag
     "@id": "#Object.u_free_psfFlux_flag"
     datatype: boolean
     description: General Failure Flag. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1171
   - name: u_fwhm
     "@id": "#Object.u_fwhm"
     datatype: double
     description: HSM moments. Measured on u-band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 1172
```

Comment:
The `<band>_fwhm` columns are derived from the `shape_<xx,yy,xy>` columns and should go near them.